_example/version/version.go

```diff
@@ -9,22 +9,15 @@ import (
 )
 
 func New(c *gin.Context) (string, error) {
-	var header string
-
-	if len(c.Request.Header["Accept"]) > 0 {
```

Comment:
If `len(c.Request.Header["Accept"])` == 0, `c.Request.Header.Get("Accept")` return "".
So `out of index` error does not occur.